python/ray/autoscaler/aws/development-example.yaml

```diff
@@ -22,6 +22,7 @@ idle_timeout_minutes: 5
 provider:
     type: aws
     region: us-west-2
+    availability-zone: us-west-2a
```

Comment:
Can we get rid of this line? I'd prefer to not specify the availability zone in the development example (if only to verify that it works without specifying it).

Reply:
Getting rid of the line results in an error parsing the YAML. I can re-implement the "default" option, or I can modify the YAML parser to make "availability-zone" optional.

Reply:
Yes, this is the bug reported in #1407. The availability zone should not be required (and the user shouldn't have to say say `default` either).